kcidb/db/postgresql/v04_09.py

```diff
@@ -0,0 +1,134 @@
+"""Kernel CI report database - PostgreSQL schema v4.9"""
+
+from kcidb.misc import merge_dicts
+import kcidb.io as io
+from kcidb.db.postgresql.schema import \
+    TextColumn, TextArrayColumn, JSONColumn, \
+    BoolColumn, TimestampColumn, Table, Index
+from .v04_08 import Schema as PreviousSchema
+
+
+# It's OK, pylint: disable=too-many-ancestors
+class Schema(PreviousSchema):
+    """PostgreSQL database schema v4.9"""
+
+    # The schema's version.
+    version = (4, 9)
+    # The I/O schema the database schema supports
+    io = io.schema.V4_5
+
+    # A map of table names and Table constructor arguments
+    # For use by descendants
+    # Add environment_compatible, and number_* fields
+    TABLES_ARGS = merge_dicts(
+        PreviousSchema.TABLES_ARGS,
+        checkouts=merge_dicts(
+            PreviousSchema.TABLES_ARGS["checkouts"],
+            columns=merge_dicts(
+                PreviousSchema.TABLES_ARGS["checkouts"]["columns"],
+                {
+                    "git_commit_tags": TextArrayColumn(),
+                    "git_commit_message": TextColumn(),
+                    "git_repository_branch_tip": BoolColumn(),
+                }
+            )
+        )
+    )
+
+    # A map of table names and schemas
+    TABLES = {
+        name: Table(**args) for name, args in TABLES_ARGS.items()
+    }
+
+    # A map of index names and schemas
+    INDEXES = merge_dicts(PreviousSchema.INDEXES, dict(
+        checkouts_git_commit_tags=Index(
+            "checkouts", ["git_commit_tags"]
+        ),
```

Comment:
I don't think the default BTREE index type would be useful here. I would say `GIN` would be more useful, similarly to `environment_compatible` in `v04_08.py`

Reply:
Right. I'll fix it.